datasync/chaindatafetcher/kafka/kafka_test.go

```diff
@@ -141,14 +166,19 @@ func (s *KafkaSuite) TestKafka_CreateAndDeleteTopic() {
 }
 
 type kafkaData struct {
-	Data []byte `json:"data"`
+	Number int
```

Comment:
Why did you add `Number`?

Reply:
This is for testing purpose. I wanted to check whether or not the segments are combined well with the given key and given producerId. I put `Number` because it has `Key()` method.

Reply:
Okay. Thanks